experiment/autobumper/bumper/bumper.go

```diff
@@ -555,24 +527,38 @@ func HasChanges() (bool, error) {
 	return len(strings.TrimSuffix(string(combinedOutput), "\n")) > 0, nil
 }
 
-func makeCommitSummary(images map[string]string) (string, error) {
-	version, err := getNewProwVersion(images)
-	if err != nil {
-		return "", err
+func getPrefixesString(prefixes []Prefix) string {
+	var prefixStrings []string
+	for _, prefix := range prefixes {
+		prefixStrings = append(prefixStrings, prefix.Prefix)
 	}
-	return fmt.Sprintf("Update prow to %s, and other images as necessary.", version), nil
+	return strings.Join(prefixStrings, ", ")
+}
+
+func makeCommitSummary(prefixes []Prefix, versions map[string][]string, consistency bool) string {
+	if !consistency || len(versions) == 0 {
+		return fmt.Sprintf("Update %s images as necessary", getPrefixesString(prefixes))
+	}
+	res := fmt.Sprintf("Update")
+	for _, prefix := range prefixes {
+		for tag, imageList := range versions {
+			//Should not be possible for tag to be in map with empty imageList
+			if strings.HasPrefix(imageList[0], prefix.Prefix) {
+				res = fmt.Sprintf("%s %s to %s,", res, prefix.Prefix, tag)
```

Comment:
This only breaks out of the inner loop correct? So the result might change depending on the order of prefixes. Is that intended?

Reply:
I believe so. This is supposed to generate a string with all of the prefixes that have been bumped and what they have been bumped to. The order of the prefixes should not matter in the commit summary. Is that what you meant?